client/CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing
+
+Client is built in `Rust 1.38.0-nightly` and compiled to WASM using `wasm-pack` tools.
+
+## Setup
+`nix-shell` sets the environment for development and tests. Type `nix-shell` from project's root directory to set up dev environment, then cd to `client` folder.
+
+## Build
+Build from source with `wasm-pack build`.
```

Comment:
FYI: We run `wasm-pack build` twice so that we have an implementation that works for both node and web.  It makes usage testing in nodejs easy to implement which doubles as validated user documentation.

See [build.sh](https://github.com/Holo-Host/chaperone-key-manager/blob/develop/build.sh) from Chaperone Key Manager.

Reply:
Implemented by Perry